### PR TITLE
Update 2b

### PR DESCRIPTION
**Pull request by manumazvi**

### CODE REVIEW

File: Assignment2/manumazvi-fromJSONtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{	
+	"@context":{
+		"vocab":"http://example.com/semweb/",
+		"name":"vocab:name",
+		"hometown":"vocab:hometown",
+		"age":"vocab:age",
+		"almaMater":"vocab:almaMater",
+		"parentsName":"vocab:parentsName",
+		"subjects":"vocab:subjects",
+		"name":"subjects:name",
```
Comment (paoespinozarias) [inline Assignment2/manumazvi-fromJSONtoJSON-LD.jsonld:10]: You have a duplicate name. Rename it, maybe as "subject":"vocab:subject" according to the name provided at the structure of your subject's array.

File: Assignment2/manumazvi-fromJSONtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		"hometown":"vocab:hometown",
+		"age":"vocab:age",
+		"almaMater":"vocab:almaMater",
+		"parentsName":"vocab:parentsName",
+		"subjects":"vocab:subjects",
+		"name":"subjects:name",
+		"teachers":"subjects:teachers",
+		"description":"subjects:description"
+	},
+	"@id":" http://example.com/semweb/people/",
```
Comment (paoespinozarias) [inline Assignment2/manumazvi-fromJSONtoJSON-LD.jsonld:14]: delete blank space " h

File: Assignment2/manumazvi-fromJSONtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	"@type":"vocab:people",
+	"name":"Manuel Mazarro Vicente",
+	"homeTown":"Puertollano",
+	"age":"22",
+	"almaMater":"UPM",
+	"parentsName": ["Manuel","Encarnacion"],
+	"subjets":
```
Comment (paoespinozarias) [inline Assignment2/manumazvi-fromJSONtoJSON-LD.jsonld:21]: Typo, it should be "subjects"

File: Assignment2/manumazvi-fromRDFtoJSON-LD.jsonld [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{
+  "@context": {
+    "prefix": <http://example.org/elements>,
+    "xsd": <http://www.w3.org/2001/XMLSchema#>
```
Comment (paoespinozarias) [inline Assignment2/manumazvi-fromRDFtoJSON-LD.jsonld:4]: They shouldn't be defined between <>
Also, there is something missing at the end of the URI of "prefix"

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

**Comment by paoespinozarias** [comment]
Requested changes on Assignment 2b are still pending. Meanwhile, @ocorcho can review Assignment 3

**Comment by paoespinozarias** [comment]
Requested changes on Assignment 2b. In the meantime, @ocorcho and @dachafra can review Assignment 3 and Assignment 4

**Comment by ocorcho** [comment]
Assignment 3 is ok

**Comment by ocorcho** [comment]
@dachafra Check this assignment to see if it can be accepted and closed.

**Comment by dachafra** [comment]
sorry! A4 is okay, merging...

**Pull request merged**